src/atoms/RouteBonus.jsx

```diff
@@ -2,26 +2,29 @@ import React from "react";
 
 import Color from "../data/Color";
 
-const RouteBonus = ({ value, size }) => {
+const RouteBonus = ({ value, size, fillColor, strokeColor, textColor }) => {
   size = size || 14;
   let width = 5.0 * value.length;
   let height = size + 6;
+  fillColor = fillColor || "white";
+  strokeColor = strokeColor || "black";
+  textColor = textColor || "black";
 
   return (
     <Color>
       {(c,t,s,p) => (
         <g>
           <polygon
             points={`${-width - 10},0 ${-width},${height*0.5} ${width},${height*0.5} ${width+10},0 ${width},${height*-0.5} ${-width},${height*-0.5}`}
-            fill={p("white")}
-            stroke={p("black")}
+            fill={fillColor}
+            stroke={strokeColor}
             strokeWidth="1"
           />
           <text
             fontWeight="bold"
             fontSize={size}
             fontFamily="sans-serif"
-            fill={p("black")}
+            fill={textColor}
```

Comment:
```suggestion
            fill={c(textColor)}
```

Reply:
Done. I also changed the atom example (no need to thave 3 of them).